dist/datasource.js

```diff
@@ -75,92 +78,36 @@ System.register(['lodash', 'moment'], function(exports_1) {
-                    // '_contentType=HostMetrics metric=CPU_LoadAvg_1Min' this
-                    // will ask the autocomplete endpoint for all values for
-                    // key '_sourceCategory' by constructing the following
-                    // autocomplete query:
-                    //
-                    //  _contentType=HostMetrics metric=CPU_LoadAvg_1Min _sourceCategory=
-                    //
-                    // We also need to tell the autocomplete endpopint the
-                    // position of the "cursor", so it notes from where in the
-                    // query it should find completitions from. The result will
-                    // look something like this:
-                    //
-                    // {
-                    //   "queryId": 0,
-                    //   "query": "_contentType=HostMetrics metric=CPU_LoadAvg_1Min _sourceCategory=",
-                    //   "pos": 65,
-                    //   "queryStartTime": 0,
-                    //   "queryEndTime": 0,
-                    //   "suggestions": [
-                    //   {
-                    //     "sectionName": "Values",
-                    //     "highlighted": null,
-                    //     "items": [
-                    //       {
-                    //         "display": "alert",
-                    //         ...
-                    //       },
-                    //       {
-                    //         "display": "analytics",
-                    //         ...
-                    //         }
-                    //       },
-                    //       {
-                    //         "display": "attack",
-                    //         ...
-                    //       },
-                    //       ...
-                    //     ]
-                    // ],
-                    // ...
-                    // }
-                    // Create the final query with the key appended.
-                    var finalQuery = metricsSelector + " " + key + "=";
-                    var position = finalQuery.length;
                     var startTime = this.start || 0;
                     var endTime = this.end || 0;
-                    var url = '/api/v1/metrics/suggest/autocomplete';
+                    // THIS IS INTERNAL API WHICH IS NOT YET PUBLISHED
+                    // Currently (2021-10-06) we need to enable it per customer, so it should be raised to sales team
+                    var url = "/api/v1/metadataCatalog/dimensions/" + encodeURIComponent(key) + "/values/search";
                     var data = {
-                        queryId: 1,
-                        query: finalQuery,
-                        pos: position,
-                        apiVersion: "0.2.0",
-                        queryStartTime: startTime,
-                        queryEndTime: endTime,
-                        requestedSectionsAndCounts: {
-                            values: 1000
-                        }
```

Comment:
I hope you know what're doing by removing all those key/value pairs :)

Reply:
it's gonna use a completely different API endpoint that doesn't support those parameters.